### PR TITLE
Fixed typoes and indentation issues in chapters 1-5

### LINKED ISSUE #170: Configuration excersise answer is wrong
@almerrifield reported the following in https://github.com/ESMValGroup/ESMValTool/issues/1923

> When uncommenting the rootpath and drs to input data, the tab structure needs to be maintained. In the example, it is correct. However, the tabs are missing in the solution for "set the correct rootpath". The recipe also will not run without the correct tab structure...

### PR DESCRIPTION
**Pull request by bvreede**
This PR fixes issue #170 by correcting the indentation in an answer block, and also fixes a number of small typoes, linting, and formatting issues in the first few chapters.

## Tasks

- [x] Give this pull request a descriptive title.
- [ ] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [ ] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #170

### CODE REVIEW

File: _episodes/03-configuration.md [hunk 1]
```diff
@@ -236,10 +241,10 @@ information about ``drs``, you can visit the ESMValTool documentation on
 > ## Explain the default drs (if working on local machine)
 >
 > 1. In the previous exercise, we set the `drs` of CMIP5 data to `default`.
-> Can you explain why?
->
+>    Can you explain why?
+
```
Comment (Peter9192) [inline _episodes/03-configuration.md:245]: Doesn't this white line break the block?

**Review by Peter9192 — COMMENTED** [review]
Looks good, just one small question

File: _episodes/03-configuration.md (hunk 1 above)
Comment (bvreede) [reply _episodes/03-configuration.md]: Oops. Well spotted.

**Review by Peter9192 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**